### ISSUE TITLE
SimPEG newsletters 

### ISSUE THREAD

**Issue by lheagy** [simpeg/community#10]
In the [2021-06-16 meeting](https://curvenote.com/@simpeg/meeting-notes/2021-06-16/), we also talked about sending out a newsletter with recent updates, releases etc. Here is a suggested format
- send out once per month, one week prior to the SimPEG seminar (#9)
- template:
    - highlights of recent SimPEG developments (e.g. new features, user-facing improvements, and any highlights from recent meetings). ~3-5 bullet points with links as appropriate 
    - announce SimPEG seminar 
    - [as appropriate] Community highlights: recent papers, upcoming talks in other forums, etc 

Todos: 
- [x] set up SendGrid (@lheagy) for the mail outs
- [x] identify someone to lead these mail outs each month

**Comment by prisae** [comment]
I suggest to also post it (1:1) to a dedicated #newsletter channel on Slack. So folks on Slack do not have to sign up for another channel of info (or everyone has the choice to either sign up to the newsletter, or join the newsletter channel).

**Comment by lheagy** [comment]
do you know how to integrate slack and sendgrid?

**Comment by prisae** [comment]
No. Another idea: It looks like you can setup a forwarding-email address that is then sent to your Slack: https://slack.com/intl/en-nl/slack-tips/send-email-to-slack

**Comment by prisae** [comment]
https://slack.com/intl/en-nl/help/articles/206819278-Send-emails-to-Slack#h_01F4WDZG8RTCTNAMR4KJ7D419V
